sources/osgShadow/ShadowMap.js

```diff
@@ -95,6 +95,7 @@ var ShadowMap = function ( settings ) {
     this._casterStateSet.addUniform( Uniform.createFloat1( 0, 'exponent1' ) );
     this._casterStateSet.addUniform( Uniform.createFloat1( 0.005, 'bias' ) );
     this._casterStateSet.addUniform( Uniform.createFloat1( 1.0 / this._textureSize, 'texelSize' ) );
+    this._casterStateSet.addUniform( Uniform.createFloat2( [ this._textureSize, this._textureSize ], 'RenderSize' ) );
```

Comment:
sadly, it's an historic one from composer.js
Whole osgjs composer and application using it uses "RenderSize"